src/driver.ts

```diff
@@ -439,7 +439,7 @@ export abstract class CMakeDriver implements vscode.Disposable {
     const candidates = this.getPreferredGenerators();
     for (const gen of candidates) {
       const gen_name = gen.name;
-      const generator_present = await (async(): Promise<boolean> => {
+      const generator_present = await(async(): Promise<boolean> => {
```

Comment:
Not a problem, but a notice tangentially related to this whitespace change.: I added a `.clang-format` for the codebase. The latest Clang releases have _excellent_ support for TypeScript and I've been running code through it as I go. In the future, it would be nice to ensure we always run formatting before commits.

Reply:
I think I have clang-format running? The editor formatted it this way when I saved the file.